models/Image.js

```diff
@@ -12,7 +12,7 @@ var Image = new keystone.List('Image', {
 
 const bucket = config['options']['gcs config']['bucket'];
 
-const resizedOpts = [['desktop', 2000], ['tablet', 1200], ['mobile', 800], ['w400', 400], ['tiny', 150]];
```

Comment:
Aside from publishing the mobile-event first, which I don't think would have explicit effect on the thumbnail resizing speed, is that any reason to reorder the items?

Reply:
No special reason. Just want mobile one to be the first sent to queue.